backend/FwLite/LcmCrdt/Changes/ExampleSentences/AddTranslationChange.cs

```diff
@@ -10,7 +10,8 @@ public class AddTranslationChange(Guid entityId, Translation translation) : Edit
 
     public override ValueTask ApplyChange(ExampleSentence entity, IChangeContext context)
     {
-        if (entity.Translations.Any(t => t.Id == Translation.Id)) return ValueTask.CompletedTask;
+        if (entity.Translations.Any(t => t.Id == Translation.Id))
+            throw new InvalidOperationException($"Translation with ID {Translation.Id} already exists in ExampleSentence with ID {entity.Id}");
```

Comment:
we really don't want to throw exceptions in an apply. If this were to get hit then the user would be stuck and the only fix (outside of modifying history) would be to change this code to not throw. If you want to use a debug assert that would be fine, otherwise we should try to handle this gracefully. In this case maybe it should replace the existing translation?

Reply:
OK, I'll add a debug.assert and just remove any existing translations.